packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -64,23 +68,27 @@ class _GestureArenaState {
 class GestureArena {
```

Comment:
Yeah, let me try that in a separate patch.  We should at least link to the web page.